# Jetson nano pinout configuration parameters

# motor control pins on Arduino
DB1: 4                     # Driver Board #1 INH
M1: 5                      # DB #1 IN1 & IN2
M2: 6                      # DB #1 IN3 & IN4
DB2: 8                     # DB #2 INH
M3: 9                      # DB #2 IN1 & IN2
M4: 10                     # DB #2 IN3 & IN4

# rgb pins
BUZ: 36
RGB_SDI: 37
RGB_RCLK: 38
RGB_SRCLK: 40
